compiler/one-cmds/one-prepare-venv

```diff
@@ -64,7 +64,7 @@ VER_ONNX=1.16.0
 VER_ONNXRUNTIME=1.18.0
 VER_ONNX_TF=1.10.0
 VER_PYDOT=1.4.2
-VER_TORCH="2.1.2+cpu"
+VER_TORCH="2.5.1+cpu"
```

Comment:
@seanshpark  

Seems that current tensorflow 2.12.1 is not compatible with this torch version. The least compatible version of tensorflow is 2.15.1.

Is there any plan to upgrade the tensorflow version?

Reply:
Yes, I'm planning to upgrade to `2.19.0`
plz check https://github.com/Samsung/ONE/issues/15226#issuecomment-2829062592 `U22.04` values.